sw/host/tests/chip/jtag/src/main.rs

```diff
@@ -0,0 +1,39 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+use anyhow::Result;
+use lazy_static::lazy_static;
+use std::collections::HashMap;
+use std::time::Duration;
+use structopt::StructOpt;
+
+use opentitanlib::app::TransportWrapper;
+use opentitanlib::io::gpio::PinMode;
+use opentitanlib::io::jtag::{Jtag, JtagParams};
+use opentitanlib::test_utils::init::InitializeTest;
+use opentitanlib::{collection, execute_test};
+
+#[derive(Debug, StructOpt)]
+struct Opts {
+    #[structopt(flatten)]
+    init: InitializeTest,
+    #[structopt(flatten)]
+    jtag: JtagParams
+}
+
+fn test_openocd(opts: &Opts, transport: &TransportWrapper) -> Result<()> {
+    let jtag = transport.jtag(&opts.jtag)?;
+    jtag.connect()?;
```

Comment:
This is a good place for context, e.g.
```suggestion
    jtag.connect().context("failed to connect JTAG")?;
```
helps a lot with debugging.

Reply:
Good point.